react-assignment1/src/components/ProfileCard/ProfileCard.css

```
@@ -0,0 +1,8 @@
+.card{
+    border: 1px solid gray;
+}
+
+.img{
```

use better classnames that are different than html tag names
```suggestion
.img {
```